cpp/react-native-hashing.cpp

```diff
@@ -23,8 +24,26 @@ namespace RNHashing {
             return jsi::Value(rt, jsi::String::createFromUtf8(rt, converter(input)));
         }
                         );
+        auto md5 = jsi::Function::createFromHostFunction(rt,
+                                                         jsi::PropNameID::forAscii(rt, "md5"),
+                                                         1,
+                                                         [](jsi::Runtime &rt, const jsi::Value &thisValue, const jsi::Value *args, size_t count) -> jsi::Value {
+            if (count == 0 || count > 1) {
+                jsi::detail::throwJSError(rt, "md5 requires one arguement");
+                return {};
+            }
+            if (!args[0].isString()) {
+                jsi::detail::throwJSError(rt, "arguement for md5 needs to be of type string");
+                return {};
```

Comment:
You can instead return undefined or null jsi value. 

return jsi::Value::undefined();

Reply:
`return {}` and `return jsi::Value::undefined()` in this case. https://stackoverflow.com/questions/39487065/what-does-return-statement-mean-in-c11

Reply:
For the sake of not having to deal with weird issues down the line (if they change the constructor or some weird unknown change happens to how the default constructor works), i changed it to ```jsi::Value::undefined()``` in #697f1c5